packages/markdown-it-plugins/tests/index.ts

```diff
@@ -0,0 +1,30 @@
+import markdownit from "markdown-it";
+import observablePlugin from "@hpcc-js/markdown-it-plugins";
+
+import "./style.css";
+
+const md = markdownit({
+    html: true,
+    linkify: true,
+    typographer: true
+});
+
+md.use(observablePlugin);
+
+const minimalMd = fetch("/tests/Basic usage/Minimal Example.md").then(response => {
+    return response.text();
+}).catch(e => {
+    console.error("Error: ", e.message ?? e);
+});
+const minimalMdText = await minimalMd;
+
+const minimalHtml = md.render("" + await minimalMd);
+// expect(minimalHtml).to.be.a.string;
```

Comment:
some commented out code in here, looked like it was moved into the index.browser.spec.ts

Reply:
Yea I will remove comment code, it was actually copied from the spec file to here...